src/app/components/wrapper-components/demo-card-wrapper/demo-card-wrapper.component.spec.ts

```diff
@@ -39,13 +49,14 @@ describe('DemoCardComponent', () => {
         // Then it should display the game
         const game: DebugElement = testUtils.findElement('app-p4');
         expect(game).withContext('game component should be displayed').toBeTruthy();
-        // from the point of view of the current player, with interactivity on
+        // from the point of view of the current player, with interactivity off
         const gameComponent: AbstractGameComponent = testUtils.getComponent().gameComponent;
-        expect(gameComponent.getPointOfView()).toBe(Player.ONE);
-        expect(gameComponent.isPlayerTurn()).toBeTrue();
-        expect(gameComponent['isInteractive']).toBeTrue();
-
+        // TODO FOR REVIEW: on fait quoi là, le point de vue doit être zero si c'est personne mais que ferions nous proporement pour dire "inactif mais pour un?"
```

Comment:
interactif du point de vue de un non ? Comme quand on joue en ligne et que t'es P1 mais que c'est pas ton tour

Reply:
Tickettons cette adaptation, car là même le test mentent !
car non on veut clairement pas "display the game interactively from the point of view the current player"

Reply:
Ok j'te laisse nommer le ticket !

Reply:
https://trello.com/c/OIL0kOvC/747-fix-interactivity-for-tutorial-where-you-dont-play-current-player?filter=notDue:true

check si il manque un truc !